src/components/Tabs/Annotation/CreateLabelForm.tsx

```diff
@@ -1,13 +1,14 @@
 import { Color as AntdColor } from "@rc-component/color-picker";
-import { Button, ColorPicker, ConfigProvider, Input, InputRef } from "antd";
-import React, { ReactElement, useEffect, useRef, useState } from "react";
+import { Button, Checkbox, ColorPicker, ConfigProvider, Input, InputRef, Radio } from "antd";
```

Comment:
![image](https://github.com/user-attachments/assets/07c0b515-73e9-4d8a-9111-14a48c32cdc9)

Updates the form to include a type and auto-increment checkbox.